CRM/Contribute/BAO/Product.php

```diff
@@ -90,7 +91,7 @@ public static function create($params) {
       ];
```

Comment:
I know this is a pre-existing condition and not a blocker for this PR, but I really hate this array of defaults. They should be set in the schema. Also, `is_active` should default to TRUE for consistency with every other entity.